gnark-utils/lib/circuit.go

```diff
@@ -8,18 +8,18 @@ import (
 	"math/big"
 
 	"github.com/consensys/gnark/frontend"
+	gl "github.com/succinctlabs/gnark-plonky2-verifier/goldilocks"
 	"github.com/succinctlabs/gnark-plonky2-verifier/types"
 	"github.com/succinctlabs/gnark-plonky2-verifier/variables"
 	"github.com/succinctlabs/gnark-plonky2-verifier/verifier"
-	gl "github.com/succinctlabs/gnark-plonky2-verifier/goldilocks"
 )
 
 type VerifierCircuit struct {
 	// A digest of the plonky2x circuit that is being verified.
 	VerifierDigest frontend.Variable `gnark:"verifierDigest,public"`
 
 	// The input hash is the hash of all onchain inputs into the function.
-	InputHash frontend.Variable `gnark:"inputHash,public"`
+	InputHash frontend.Variable `gnark:"inputHash"`
```

Comment:
I found that I could remove this from the function `verifyProof` of generated `verifier.sol` when deletes `public`.